warehouse/query-core/src/test/java/datawave/query/jexl/visitors/IteratorBuildingVisitorTest.java

```diff
@@ -51,7 +52,8 @@ private IteratorBuildingVisitor getDefault() {
      */
     @Test
     public void visitEqNode_nullValueTest() {
-        ASTEQNode node = (ASTEQNode) JexlNodeFactory.buildEQNode("FIELD", null);
+        ASTEQNode node = (ASTEQNode) JexlNodeFactory.buildEQNode("FIELD", "blah");
```

Comment:
The problem is that JexlNodeFactory.buildEQNode eventually calls JexlNodes.setLiteral.  If that is called with a null value, it will throw an exception for a precondition 'not null' error.  To get around that, I first create a valid EQ node with a throwaway value, then on the next line I replace the string literal child with one that has no value.  So, the tests should be completely equivalent - just the setup is different.

Reply:
That was the context I needed, perfect.